bin/fillTestData/Tribes.js

```diff
@@ -36,7 +42,7 @@ var addTribes = function (max) {
         tribe.created = Date.now();
         tribe.modified = Date.now();
         tribe.public = true;
-        // tribe.image_UUID = faker.random.uuid();
+        tribe.image_UUID = tribesData[random(tribesData.length)].image_UUID;
```

Comment:
What do you think if we only include the UUIDs as an array here, since rest of the data isn't used anyway?

```
var UUIDs = [
  "171433b0-853b-4d19-a8b4-44def956696d",
  "e69eb05f-773f-423c-9246-43629b5a8baf",
  ...
];